debugger/src/main/java/org/codice/acdebugger/api/ReflectionUtil.java

```diff
@@ -341,10 +415,7 @@ public Method findMethod(ReferenceType type, String name, String signature) {
     if (obj == null) {
       return null;
     }
-    final ReferenceType type =
-        (obj instanceof ClassObjectReference)
-            ? ((ClassObjectReference) obj).reflectedType()
-            : obj.referenceType();
+    final ReferenceType type = obj.referenceType();
```

Comment:
ℹ️ The original code was to handle static method calls which are now handled separately with `invokeStatic()`. It also created issues when actually trying to invoke methods on the class `Class`.